addons/stock/product.py

```diff
@@ -201,7 +201,22 @@ def _product_available_text(self, cr, uid, ids, field_names=None, arg=False, con
             res[product.id] = str(product.qty_available) +  _(" On Hand")
         return res
 
+    def _reorder_rule_count(self, cr, uid, ids, field_name, arg, context=None):
+        res = {}
+        for product in self.browse(cr, uid, ids, context=context):
+            no_of_rule = len(product.orderpoint_ids)
+            res[product.id] = {'reordering_count' : no_of_rule}
+            if no_of_rule == 1:
+                res[product.id].update({
+                    'reordering_min' : product.orderpoint_ids.product_min_qty,
+                    'reordering_max' : product.orderpoint_ids.product_max_qty
+                })
+        return res
+
     _columns = {
+        'reordering_count': fields.function(_reorder_rule_count, type='integer', multi=True),
+        'reordering_min': fields.function(_reorder_rule_count, type='float', multi=True),
+        'reordering_max': fields.function(_reorder_rule_count, type='float', multi=True),
```

Comment:
i think better to  give field name like 'reordering_max_qty' and 'reordering_min_qty'